package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@keycloak/keycloak-admin-client": "^26.1.4",
-    "@patternfly/react-component-groups": "6.0.0-alpha.16",
```

Comment:
Can't upgrade this out of `alpha`, as the specific component we use is only available in alpha